Neos.Neos/Tests/Behavior/Features/FrontendRouting/TetheredSiteChildDocuments.feature

```diff
@@ -0,0 +1,70 @@
+@fixtures @contentrepository
+Feature: Basic routing functionality (match & resolve document nodes in one dimension)
+
+  Background:
+    Given using no content dimensions
+    And using the following node types:
+    """yaml
+    'Neos.ContentRepository:Root': {}
+    'Neos.Neos:Sites':
+      superTypes:
+        'Neos.ContentRepository:Root': true
+    'Neos.Neos:Document':
+      properties:
+        uriPathSegment:
+          type: string
+    'Neos.Neos:Site':
+      superTypes:
+        'Neos.Neos:Document': true
+      properties:
+        uriPathSegment: ~
```

Comment:
... i meant that this doesnt work ;) its a noop ^^

Reply:
okay, removed it